backend/src/Designer/Models/ContentsListResponse.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Runtime.Serialization;
+
+namespace Altinn.Studio.Designer.Models
+{
+    [DataContract]
+    public class ContentsResponse
+    {
+        public ContentsResponse(FileLinksResponse links = default(FileLinksResponse), string content = default(string), string download_url = default(string), string encoding = default(string), string git_url = default(string), string html_url = default(string), string name = default(string),
+            string path = default(string), string sha = default(string), int size = default(int), string submodule_git_url = default(string), string target = default(string), string type = default(string), string url = default(string))
+        {
+            this.links = links;
+            this.content = content;
```

Comment:
is contentent the real file content of the json file?

Reply:
This content is the content of the repository. So it is information about the file in repo. I'll DM you an example.